src/sqlfluff/core/rules/__init__.py

```diff
@@ -1,7 +1,18 @@
-"""init py for the new rules crawlers."""
+"""Register all the rule classes with their corresponding rulesets (just std currently)."""
 
-from .std import std_rule_set
-from .base import RuleSet, rules_logger  # noqa
+import inspect
+
+from .base import RuleSet
+from .config_info import STANDARD_CONFIG_INFO_DICT
+from . import std
+
+
+std_rule_set = RuleSet(name="standard", config_info=STANDARD_CONFIG_INFO_DICT)
+
+for name, val in std.__dict__.items():
+    # Filter on the item being a rule
+    if name.startswith("Rule_L") and inspect.isclass(val):
```

Comment:
Nitpick: This code seems to overlap with code in `src/sqlfluff/core/rules/std/__init__.py`.

Reply:
Thanks, resolved!